Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -142,8 +144,8 @@ def create_issue(repo, title, body)
   def gist_logs
     gist_logs_args.parse
 
-    Install.perform_preinstall_checks(all_fatal: true)
-    Install.perform_build_from_source_checks(all_fatal: true)
+    perform_preinstall_checks(all_fatal: true)
+    perform_build_from_source_checks(all_fatal: true)
```

Comment:
The previous version reads better to me.

Reply:
The previous version will always use the global `Homebrew.args`, this will use the `args` in the current class/module. This just happens to still be `Homebrew.args` at the moment.

Reply:
Hmm, I think I'd prefer keeping it in the module and passing through `args`?

Reply:
Done.